src/style.css

```diff
@@ -0,0 +1,47 @@
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
```

Comment:
you can't set margins to BEM block
if you need to reset default styles for the tag elements, use reset.css file or reset it in the same way, as you did for the `body`